tailwind.config.js

```diff
@@ -15,4 +15,5 @@ module.exports = {
     extend: {},
   },
   plugins: [],
+  important: true,
```

Comment:
@taro04 
Did you have design problems without it? I am wondering we should do this between the following advantages and disadvantages.

- Advantages: If Angular Material and tailwind were to be used together with unintended conflicts, with this setting, the tailwind CSS would be set definitely first, and I think it might be a good solution.
- Disadvantages: I concerned this is too strong settings.

@KimuraYu45z 
Can you please advise if it should be important true or not?

Reply:
@YasunoriMATSUOKA 
This setting was necessary to delete margin of h* tag
[https://cauchye-education.slack.com/archives/CSJD5N06M/p1645783657944079](https://cauchye-education.slack.com/archives/CSJD5N06M/p1645783657944079)